opm/parser/eclipse/EclipseState/SimulationConfig/RockConfig.hpp

```diff
@@ -89,8 +89,8 @@ struct RockComp {
     std::vector<RockComp> m_comp;
     std::string num_property = ParserKeywords::ROCKOPTS::TABLE_TYPE::defaultValue;
     std::size_t num_tables = ParserKeywords::ROCKCOMP::NTROCC::defaultValue;
-    bool m_water_compaction;
-    Hysteresis hyst_mode;
+    bool m_water_compaction = false;
+    Hysteresis hyst_mode = Hysteresis::NONE;
```

Comment:
For hysteresis the default is `Hysteresis::REVERS` so I guess that could be used.